cdhweb/people/templates/people/profile_page.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'base.html' %}
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block page-subtitle %}{{ page.title }} | People | {% endblock %}
+
+{% block main %}
+<div class="profile">
+<header>
+    <h1>{{ page.title }}</h1>
+    {# display all positions, current first; dates for non-current #}
+    {% for position in page.person.positions.all %}
+        <p class="title">{% if not position.is_current %}
+            {{ position.start_date.year }}{% if position.end_date.year != position.start_date.year%}-{{ position.end_date.year }}{% endif %}
+            {% endif %}
+        {{ position.title }}
+        </p>
+    {% endfor %}
+</header>
+
+<div class="links">
+    {# NOTE equiv to contributors in project #}
+    <ul>
+    {% for resource in page.person.personresource_set.all %}
+        <li><a href="{{ resource.url }}">{{ resource.resource_type.name }}</a></li>
+    {% endfor %}
+    </ul>
+</div>
+
+{% if page.image %}
+    {% image page.image fill-790x632 as profile_img %}
```

Comment:
do we already have later tasks about making these responsive in future?

Reply:
imo would be covered by #266

Reply:
ok, fair. I added a preliminary check list to that issue so we'll remember it is included